codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/IdempotencyTokenMiddlewareGenerator.java

```diff
@@ -151,12 +159,24 @@ private MemberShape getMemberWithIdempotencyToken(Model model, OperationShape op
                             }
                             return false;
                         })
-                        .buildMiddlewareStack((writer, service, operation, protocolGenerator, stackOperand) -> {
-                            writer.write("$L.Initialize.Add(&$L{},middleware.After)",
-                                    stackOperand,
-                                    getIdempotencyTokenMiddlewareName(operation));
```

Comment:
Why remove this general capability of being able to hook into and register middleware? It seems valuable for middleware that is not drive by configuration from options to still be able to insert themselves as plugins.